integration_test/support/models.exs

```diff
@@ -115,6 +115,7 @@ defmodule Ecto.Integration.User do
     * Timestamps
     * Relationships
     * Dependent callbacks
+    * Embedding
```

Comment:
To make things simpler, what if we flip things around? Let's create an Account model that embeds User instead of User embedding Profile. There are already too many things happening in the User module.